pkg/monitoring/metrics/virt-controller/vmstats_collector_test.go

```diff
@@ -1033,3 +1190,29 @@ func expectVMs(
 	expectVM(filterResultsByVM(results, name, namespace), name, namespace)
 	expectVM(filterResultsByVM(results, name+"-2", namespace), name+"-2", namespace)
 }
+
+func expectDefaultCPUResourceRequests(crs []operatormetrics.CollectorResult) {
```

Comment:
@sradco We prefer no to use assertions in helper functions, can we please refactor this helper function to return the default CPU requests and an error, and the test itself to assert on the returned error and on the returned defaults?